ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dot1x.j2

```diff
@@ -18,8 +18,64 @@ dot1x protocol bpdu bypass
 {%     if dot1x.dynamic_authorization is arista.avd.defined(true) %}
 dot1x dynamic-authorization
 {%     endif %}
-{%     if dot1x.mac_based_authentication is arista.avd.defined or dot1x.radius_av_pair is arista.avd.defined %}
+{%     if dot1x.mac_based_authentication is arista.avd.defined or dot1x.radius_av_pair is arista.avd.defined or dot1x.aaa.unresponsive is arista.avd.defined %}
 dot1x
+{%         if dot1x.aaa.unresponsive is arista.avd.defined %}
+{%             set aaa_config = "aaa unresponsive" %}
+{%             if dot1x.aaa.unresponsive.phone_action is arista.avd.defined %}
+{%                 set aaa_config_phone = aaa_config ~ " phone action" %}
+{%                 if dot1x.aaa.unresponsive.phone_action.apply_alternate is arista.avd.defined(true) %}
```

Comment:
i believe we can optimize this nested if else conditions, lets connect via a call once you done with other comments

Reply:
Sure